packages/components/atoms/f-card/src/components/Card.vue

```diff
@@ -13,7 +13,7 @@
                 v-if="cardHeading"
                 :class="[
                     $style['c-card-heading'],
-                    (cardHeadingPosition !== 'left' ? $style[`c-card--${cardHeadingPosition}`] : '')
+                    { [$style[`c-card--${cardHeadingPosition}AlignedText`]]: cardHeadingPosition !== 'left' }
```

Comment:
If this is used just for aligning the heading, classname should be c-card-heading--alignRight, (as it's on the same class level as c-card-heading)